rio_cogeo/cogeo.py

```diff
@@ -190,18 +192,19 @@ def cog_translate(
                 max_zoom = get_max_zoom(src_dst, lat=lat, tilesize=tilesize)
 
                 extrema = tile_extrema(bounds, max_zoom)
-                w, n = mercantile.xy(
-                    *mercantile.ul(extrema["x"]["min"], extrema["y"]["min"], max_zoom)
+
+                left, _, _, top = mercantile.xy_bounds(
+                    extrema["x"]["min"], extrema["y"]["min"], max_zoom
                 )
```

Comment:
`mercantile.xy_bounds` is better than `mercantile.xy(mercantile.ul(` https://github.com/mapbox/mercantile/pull/101#issuecomment-612265337